File: Application/Features/Medias/PremieresAndAnnouncements/GetReleases/ReleaseItemResponse.cs
using Application.Features.Genres.GetGenres;
using Application.Features.Medias.Movies.Common;

namespace Application.Features.Medias.PremieresAndAnnouncements.GetReleases
{
    public record ReleaseItemResponse(
        Guid Id,
        string Title,
        string Description,
        DateTime ReleaseDate,
        string MediaType,
        GenreResponse Genre,
        MediaStatsResponse MediaStatsResponse);
}

File: angular-movieFrontend/src/Data/Response/ReleaseItemResponse.ts
import { GenreResponse } from "./GenreResponse";
import { MediaStatsResponse } from "./MediaStatsResponse";

export interface ReleaseItemResponse {
    Id:string;
    Title:string;
    Description:string;
    ReleaseDate:Date;
    MediaType:string;
    GenreResponse: GenreResponse;
    MediaStatsResponse: MediaStatsResponse;
}